### PR TITLE
Use v1 of n-ui

### PR DESCRIPTION
**Pull request by ironsidevsquincy**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
   "name": "next-signup",
   "dependencies": {
-    "n-ui": "1.0.0-beta.9",
-    "immutable": "^3.7.6",
     "Case": "^1.4.1",
-    "fetchres": "^1.5.1"
+    "fetchres": "^1.5.1",
+    "immutable": "^3.7.6",
```
Comment (lc512k) [inline bower.json]: weren't these two part of n-ui?

**Comment by lc512k** [comment]
Thanks @ironsidevsquincy! Took the liberty of merging it to get the change asap into other branches.

File: bower.json (hunk 1 above)
Comment (ironsidevsquincy) [inline bower.json]: I was just being anal and reordered these alphabetically. Doesn't look like n-ui depends on them - https://github.com/Financial-Times/n-ui/blob/master/bower.json

**Comment by ironsidevsquincy** [comment]
np. thanks

**Pull request merged**